packages/accordion/src/renders/AccordionItem.tsx

```diff
@@ -0,0 +1,14 @@
+import { PluginElementRenderProps } from '@yoopta/editor';
+
+export const AccordionItem = (props: PluginElementRenderProps) => {
+  const { element, attributes, children } = props;
+
+  return (
+    <li
+      {...attributes}
+      className="yoo-accordion-border yoo-accordion-border-[#525252] yoo-accordion-bg-[#303030] yoo-accordion-rounded-none yoo-accordion-relative yoo-accordion-w-full"
```

Comment:
wondering those two `[#525252]` is a standard coloring schema for yoopta, or/and might be re-used from somewhere or taken from Props with default values as defined in the code?